webpack.config.babel.js

```diff
@@ -136,17 +136,27 @@ const configDist = {
     libraryTarget: 'umd',
     path: `${__dirname}/dist`,
     publicPath: '/',
-    filename: 'onfido.min.js'
+    filename: 'onfido.min.js',
+    chunkFilename: '[name].bundle.js',
```

Comment:
what is `name` here?

Reply:
what is defined in the comment of `webpackChunkName`